### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSettings_ScreenReader: JAWS/NVDA does not read the combo box (i.e. off)  name and only reads it as "TAB" i.e.(combobox present against Commonly Used Files:AutoSave)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52472]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while JAWS is ON.
2. Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown-->New settings editor
3. Navigate to the combo box displayed by expanding the list items.

**Actual:**
JAWS does not read the combo box name and only reads it as "TAB" 
ScreenReader announces as "TAB" instead of announcing its name and role type.

**Expected:**
JAWS does not read the combo box name and only reads it as "TAB" 
The control should have a proper name and its name and role type should be announced by the screen reader

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to get clarity about control's name,state,role,value and other important info if only "TAB" is read by the screen reader

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_UserSettings_JAWS.pptx](https://github.com/Microsoft/vscode/files/2119417/A11y_VSCode_UserSettings_JAWS.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
Which combo box are you talking about? I don't see a combobox in the screenshot.

**Comment by auchenberg** [comment]
Closing this issue as it's only observed in JAWS, and this isn't strictly needed to meet MAS C compliance.

**Comment by AccessibilityTestingTeam-TCS** [comment]
The combo box control for which the bug is logged is present against Commonly Used Files:AutoSave i.e. off, Visual studio light etc.
![off combobox](https://user-images.githubusercontent.com/39204909/41905288-53f9c34c-7958-11e8-8d0f-a3c408afecc8.png)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52582
